ui/src/components/yaml/index.tsx

```diff
@@ -254,6 +255,8 @@ const Yaml = (props: IProps) => {
       setStreaming(false)
     }
   }
```

Comment:
Potential null reference - consider adding a null check for `contentRef.current` before accessing `getBoundingClientRect()`.